### ISSUE TITLE
Can't lift back to older assemblies

### ISSUE THREAD

**Issue by broughtonjp** [AndersenLab/liftover-utils#4]
I'm encountering a problem using liftover.py. I can't seem to convert from a new assembly to an older one (WS245 to WS220). However, when I reverse the release1 and release2 (WS220 to WS245) using the same file it seems to be functional. Am I missing something for lifting back to an older assembly?

**Comment by danielecook** [comment]
Yeah, unfortunately, right now - I don't think its possible to lift back. Only forward. I tried messing around with it for a while, but I couldn't get it to work. I'll try to update this soon making that clear. What are you using that is WS220? 

In the end - it was easier for us to move everything forward rather than move some things back.

**Comment by broughtonjp** [comment]
Thanks for the quick reply. All of our current tools are built to work with
UCSC's ce10, but we were trying to use the more up to date gene definitions
(particularly 5'UTRs and 3'UTRs) from the current releases.

- James

**Comment by danielecook** [comment]
James -

I would suggest moving over to wormbase 245 if you can. Note that they frequently update the genome (e.g. Right now I believe it is WS247) without changing coordinates. Any new data added can be used in this case.

I'd love to see ucsc update but I don't think it will happen anytime soon.
